### PR TITLE
Filter bar results

### PR DESCRIPTION
**Pull request by Matt-Warnock**
Orders bar results by our formula, based on number of user_ratings.
Filters that down to 5 results (order: first best). 

Just seen that I need to rename 'LeastRecommed' to 'mostRecomended'

### CODE REVIEW

File: src/sorter.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    const orderedBars =  this.#orderByRating(this.#addPopulerRating(goodRating));
+
+    return orderedBars.slice(-5).reverse();
+  }
+
+  #ratingFilter(results) {
+    return results.filter( bar => bar.rating > 4 )
+  }
+
+  #addPopulerRating(results) {
```
Comment (s-palmer) [inline src/sorter.js]: Can we just change the name of this method to be addPopularRating to be consistent with the others?

File: tests/sorter.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+describe('Sorter', () => {
+  const sorter = new Sorter;
+
+  describe('#topFiveBars', () => {
+    const results = response.results;
+    const topFiveResults = sorter.topFiveBars(results);
+
+    it('filters results to rating over four', () => {
+      aboveFourRating = topFiveResults.every( bar => bar.rating > 4 );
```
Comment (s-palmer) [inline tests/sorter.test.js:12]: nice test

File: tests/sorter.test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+      expect(aboveFourRating).toEqual(true);
+    })
+
+    it('adds a popular rating to each bar', () => {
+      const leastRecommended = topFiveResults[0];
+
+      expect(leastRecommended.popularRating).toEqual(53.2)
```
Comment (s-palmer) [inline tests/sorter.test.js]: LeastRecommended is the lowest scoring bar right? Maybe let's change popularRating to popularityRating instead so it's a bit clearer that it's not like leastRecommended but popular.

**Review by s-palmer — CHANGES_REQUESTED** [review]
Just a couple of small suggestions but looks good to me otherwise

File: src/sorter.js (hunk 1 above)
Comment (Matt-Warnock) [reply src/sorter.js]: Ah, its a typo, you mean?

File: tests/sorter.test.js (hunk 2 above)
Comment (Matt-Warnock) [reply tests/sorter.test.js:12]: Thanks

File: src/sorter.js (hunk 1 above)
Comment (s-palmer) [reply src/sorter.js]: Ah right yes just that

**Review by s-palmer — APPROVED** [review]
Changes look good!

**Pull request merged**